README.md

```diff
@@ -97,6 +97,9 @@ Example ZLint CLI usage:
 	echo "Lint mycert.pem with all of the lints except for ETSI ESI sourced lints"
 	zlint -excludeSources=ETSI_ESI mycert.pem
 
+	echo "List available lint profiles. A profile is a pre-defined collection of lints."
```

Comment:
I was apprehensive of adding an example of running a profile here because don't actually have any yet. That is, I didn't want to give an example that doesn't actually run.